airbyte-integrations/connectors/source-sftp-bulk/source_sftp_bulk/spec.py

```diff
@@ -48,6 +48,17 @@ class Config:
         default="/",
         pattern_descriptor="/folder_to_sync",
     )
+    use_file_transfer: bool = Field(title="File Sync (Experimental)", description="Enable file-based bulk load", default=False)
+
```

Comment:
@aldogonzalez8 - Should we drop this boolean? Or not yet?

Reply:
Yes, will just remove before merge, probably right no.

Reply:
Removed now.